static/dotbotsim.js

```diff
@@ -146,108 +146,65 @@ function drawDotBots(data) {
     dotBots
         .transition()
             .attr("cx", function(d) { return scaleFactor*d.x; })
-            .attr("cy", function(d) { return scaleFactor*d.y; });
+            .attr("cy", function(d) { return scaleFactor*d.y; })
+            .attr("fill",function(d,i) {return d.isRelay === true ? 'black' : dotBotcolors[i%10];});
```

Comment:
rather than manually changing teh color here, you can add another class "relay". In any case, why black?